src/url-params.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * preact-router does not parse "+" as spaces the way JS encodes them
+ */
+function pathWithParams(path: string, searchParams: URLSearchParams): string {
+  return `${path}?${searchParams.toString().replace(/\+/g, "%20")}`;
+}
```

Comment:
this is kind of related to our router? but not quite? so I could see an argument for putting it in there?

Reply:
I could see it going either way. Maybe even an enhancement to our `Link` wrapper like `<Link path={...} params={...} />`. Or we could go full-"one package per method" and it could be a standalone package via [npm workspaces](https://docs.npmjs.com/cli/v7/using-npm/workspaces) 😄 !

Reply:
Or maybe, with #29, this could be a responsibility of `ReportFilterContextProvider` ? Passing a method like `setParameters` to context, which creates the URL to route when set. Then we render a `<button class="usa-button usa-button--unstyled">` instead of a `<Link>`.

Reply:
good idea, added in 67c45ab6f761d3603fe95cfaeb370d11c65466bb